package.json

```diff
@@ -2,16 +2,23 @@
   "name": "polygonize",
   "version": "1.0.3",
   "description": "Javascript implementation of GEOS's Polygonize function",
-  "main": "dist/index.js",
+  "main": "cjs/index.js",
+  "module": "es/index.js",
```

Comment:
Why build for ES Module? You could simply use your `src` and it would be valid.

No need to compile with Babel.

Up to you, this will still work, but when testing the line mapping might get a bit confusing since it will now hit the compile Babel code instead of your source code.

Reply:
I thought that Turf needed the traspiled code, my bad. 

I'll use the `src/index.js` for `module` and the transpiled es5 for `main`

Reply:
You are correct, I do remember asking that in the past, however for the `module` can be written in ES6, it's only the `main` that would need to be pure CommonJS.

Question: does your code work with Node4? That's the ideal goal for acceptable syntax, for now are test suite is only supported in Node6+

Reply:
Yes, the code runs in node 4 (using the `@std/esm` for module importing). The only file that doesn't work is the main test (due to the use of destructuring assignment).

I've removed the babel compiled es module, and remove the `src` folder in the `npmignore` file.